tests/unit/test_urls.py

```diff
@@ -203,7 +203,7 @@ def test_contact(self):
     def test_activity_page_401_if_not_logged_in(self):
         """Test that asks user to log in before displaying activity."""
         rv = self.app.get('/me')
-        self.assertEqual(rv.status_code, 401)
+        self.assertEqual(rv.status_code, 404)
```

Comment:
this test is not necessary anymore as the route has been removed. And like any other random strings it will return a 404.